.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: objective-c
+osx_image: xcode7.3
+before_install:
+  - brew update
+  - brew outdated xctool || brew upgrade xctool
+  - brew outdated carthage || brew upgrade carthage
+before_script:
+  - carthage bootstrap --no-build
+script:
+  - xctool -workspace Postal.xcworkspace -scheme Postal-iOS -sdk iphonesimulator9.3 build
+  - xctool -workspace Postal.xcworkspace -scheme Postal-macOS -sdk macosx10.11 build
```

Comment:
@jeremiegirault As you can see the CI will only build the project to valide the PR. We don't execute tests because of the dependence of the file containing credentials of different accounts. Of course we can add an encrypted file as explained here https://docs.travis-ci.com/user/encrypting-files/ but it will require an extra amount of work. 
I should be sufficient for now to test future PR. And we'll create another PR to handle tests correctly in the CI.  WDYT?